astro/src/content/blog/announcing-fusionauth-1-50.mdx

```diff
@@ -0,0 +1,122 @@
+---
+publish_date: 2024-04-24
+title: Announcing FusionAuth 1.50
```

Comment:
@mooreds Would you be opposed to changing this to include the OAuth Scopes inclusion? Perhaps "Announcing FusionAuth 1.50 - OAuth Scopes are Here"